webapp/src/blocks/card.ts

```diff
@@ -36,6 +37,7 @@ function createCard(block?: Block): Card {
             contentOrder,
             isTemplate: block?.fields.isTemplate || false,
         },
+        limited: false,
```

Comment:
I don't think we want to default it to false, right? It should just be passed through? https://github.com/mattermost/focalboard/blob/a863e43fb0876d21d7996d92f5c3452a99c25437/webapp/src/blocks/block.ts#L63

similar to my comment above, could this be moved to `createBlock` instead of being in `createCard`?

Reply:
Yes. Fixed